src/components/SelectInput/SelectInput.scss

```diff
@@ -1,3 +1,5 @@
+/* The following styles required the :global tag since they need to be applied to
+react-select elements for which we do not own the markup */
 :global {
   .selectInputWrapper {
```

Comment:
:global means nothing outside of a module.

I would make this a module and reference the globals like so:
```
.selectInputWrapper {
  :global {
    .reactStyle {
       ...
    }
  }
}
```

Reply:
@RyanMacBern due to the way react-select is styled, where we have to override classes on elements we don't render ourselves, having these inside of a module, with :global or not, broke that implemenation. We can perhaps continue experimenting as to why that happens but at the moment this seems to be the best approach.

Reply:
I think the problem is likely because you have a non-global (selectInputWrapper) mixed with the globals. My example should fix that.